.github/workflows/tests.yaml

```diff
@@ -478,9 +478,29 @@ jobs:
           # We're currently including `nightly` because I'm not sure whether
           # it's always reliable; e.g. `cargo-audit`
           allowed-failures: check-links-markdown, nightly
-          # Easier than writing out every job — and it's our own rules deciding
-          # whether jobs are skipped or not.
-          allowed-skips: ${{ toJSON(needs) }}
+          # We skip jobs deliberately, so we are OK if any are skipped.
+          #
+          # Copy-pasted from `needs`, since it needs to be a json list, so `${{
+          # toJSON(needs) }}` (which is a map) doesn't work.
+          # https://github.com/re-actors/alls-green/issues/23
+          allowed-skips: |
+            [
```

Comment:
FYI, you don't have to wrap this as a JSON list, a comma-separated list works too:
https://github.com/python/cpython/blob/809ea7c/.github/workflows/build.yml#L550-L577